### PR TITLE
Document solution to migration name collision

### PR DESCRIPTION
**Pull request by bjuppa**
This PR adds a Troubleshooting section to the readme, and explains a solution to a problem that can occur if running both normal testbench tests and Dusk tests in the same test suite.

References #26

### CODE REVIEW

**Comment by crynobone** [comment]
Thank you

File: README.md [hunk 1]
```diff
@@ -162,3 +163,35 @@ Run only your browser tests by running phpunit with the `--testsuite=Browser` op
...
+*
+* Overrides \Orchestra\Testbench\Dusk\TestCase::getBasePath
+*       and \Orchestra\Testbench\Concerns\CreatesApplication::getBasePath
+*
+* @return string
+*/
+protected function getBasePath()
+{
+    // Adjust this path depending on where your override is located.
+    return __DIR__.'/../vendor/orchestra/testbench-dusk/laravel';
```
Comment (crynobone) [inline README.md]: @Keoghan is there a way to write this with the help of `konsulting/project-root`?

Comment (Keoghan) [reply README.md]: I think with the following: 

```php 
return \Konsulting\ProjectRoot::forPackage('testbench-dusk')->resolve(__DIR__).'/laravel';
```

Will see how that behaves in a real app though, and check back.

Comment (Keoghan) [reply README.md]: Thinking I might be able to add a utility to project root that resolves the path for the first of a set of packages, if that's of interest here.

Comment (crynobone) [reply README.md]: I have to see the proposed code first. No idea at the moment. Would be easy if we can do something like this.

```php
return Finder::package('testbench-dusk', __DIR__)->find('laravel');
```

**Pull request merged**